Share/NCShareExtension.swift

```diff
@@ -370,6 +375,9 @@ extension NCShareExtension {
         // CHUNCK
         metadata.chunk = chunckSize != 0 && metadata.size > chunckSize
 
+        let status = NSLocalizedString("_upload_file_", comment: "") + " \(counterUploaded + 1) " + NSLocalizedString("_of_", comment: "") + " \(filesName.count)"
+        IHProgressHUD.set(status: status)
+        
         NCNetworking.shared.upload(metadata: metadata) { } completion: { errorCode, _ in
```

Comment:
Hmm.. I thought about that too.. but this will also reset the progress, right? And it causes the progress to glitch.. Maybe we can call it even before that..

![ScreenRecording2](https://user-images.githubusercontent.com/18512366/150341374-2fe2f16b-328f-445e-b8e5-f0bae12df7cc.mov)